.ruby-version

```diff
@@ -0,0 +1 @@
+2.4.1
```

Comment:
Unfortunately, we aren't ready to make this jump yet. We are running 2.2.6. Is 2.4.1 necessary for something you are working on or can we lock it to 2.2.6?